newsroom/template_filters.py

```diff
@@ -38,7 +38,7 @@ def datetime_long(datetime):
 
 
 def date_header(datetime):
-    return format_datetime(parse_date(datetime if datetime else utcnow()), 'EEEE, MMMM d, yyyy')
+    return format_datetime(parse_date(datetime if datetime else utcnow()), 'EEEE, dd.MM.yyyy')
```

Comment:
This format should really be in a config somewhere

Reply:
What about this one? A simple config with default in `newsroom/web/default_settings.py` should suffice